lib/include/jxl/decode.h

```diff
@@ -238,6 +243,39 @@ typedef enum {
    * image and always before JXL_DEC_FULL_IMAGE.
    */
   JXL_DEC_JPEG_RECONSTRUCTION = 0x2000,
+
+  /** Informative event by JxlDecoderProcessInput: The header of a box of the
+   * BMFF container format is decoded. The following API functions related to
```

Comment:
nit: "Box Metadata File Format container format" is a bit redundant :)

Reply:
Fixed, but leaving in the term BMFF for reference